### PR TITLE
magento/devdocs#: added example of using the gallery widget

### PR DESCRIPTION
**Pull request by DashukVolodymyr**
## Purpose of this pull request

This pull request (PR) adds an example of using the gallery widget.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/widget_gallery.html

whatsnew
Added an example use case of the [gallery widget](https://devdocs.magento.com/guides/v2.4/javascript-dev-guide/widgets/widget_gallery.html).

### CODE REVIEW

File: src/guides/v2.3/javascript-dev-guide/widgets/widget_gallery.md [hunk 1]
```diff
@@ -580,3 +580,56 @@ api.updateOptions([{
 [Fotorama widget]: http://fotorama.io/
 [lib/web/mage/gallery/gallery.js]: {{ site.mage2bloburl }}/{{ page.guide_version }}/lib/web/mage/gallery/gallery.js
 [lib/web/magnifier/magnify.js]: {{ site.mage2bloburl }}/{{ page.guide_version }}/lib/web/magnifier/magnify.js
+
+## Code sample
+
+This example shows a use case for a gallery widget on any page.
```
Comment (BarnyShergold) [inline src/guides/v2.3/javascript-dev-guide/widgets/widget_gallery.md:586]:
```suggestion
This example shows a use case for a gallery widget on any page:
```

File: src/guides/v2.3/javascript-dev-guide/widgets/widget_gallery.md [hunk 2]
```diff
@@ -580,3 +580,56 @@ api.updateOptions([{
...
+                    }
+                }
+            }, element);
+        });
+    });
+});
+</script>
+```
+
+The breakpoints options are set in the `view.xml` configuration file of a theme. The file is conventionally located in `<theme_dir>/etc`.
```
Comment (BarnyShergold) [inline src/guides/v2.3/javascript-dev-guide/widgets/widget_gallery.md:627]:
```suggestion
The breakpoints options are set in the `view.xml` configuration file of a theme. The file is located in `<theme_dir>/etc`.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
NIce addition

File: src/guides/v2.3/javascript-dev-guide/widgets/widget_gallery.md (hunk 1 above)
Comment (DashukVolodymyr) [reply src/guides/v2.3/javascript-dev-guide/widgets/widget_gallery.md]: Hi @BarnyShergold 
All your suggestions have been implemented!

Thank you!

File: src/guides/v2.3/javascript-dev-guide/widgets/widget_gallery.md (hunk 2 above)
Comment (DashukVolodymyr) [reply src/guides/v2.3/javascript-dev-guide/widgets/widget_gallery.md]: Hi @BarnyShergold 
All your suggestions have been implemented!

Could you please verify this PR?

Thank you!

**Review by BarnyShergold — APPROVED** [review]
Nice work!

**Review by keharper — APPROVED** [review]

**Comment by keharper** [comment]
running tests

**Pull request merged**